YoCode/Output/WebWriter.cs

```diff
@@ -24,8 +24,7 @@ public WebWriter()
 
         public void AddErrs(IEnumerable<string> errs)
         {
-            errors.Append(WebElementBuilder.FormatAccordionElement(WebElementBuilder.FormaFeatureTitle("Errors present",false),
-                WebElementBuilder.FormatListOfStrings(errs)));
+            /* HTML does not support error output */
```

Comment:
This is a code smell called Refused Bequest. It's an odd name but don't blame me, I didn't invent it!
https://refactoring.guru/smells/refused-bequest
It basically means this method should probably not be on the `IPrint` interface if only one of the derived classes use it.